Unity-MCP-Plugin/Assets/root/Editor/UI/uxml/MCPToolsWindow.uxml

```diff
@@ -0,0 +1,19 @@
+<ui:UXML xmlns:ui="UnityEngine.UIElements" xmlns:uie="UnityEditor.UIElements" editor-extension-mode="False">
+    <Style src="project://database/Assets/root/Editor/UI/uss/MCPToolsWindow.uss?fileID=7433441132597879392&amp;guid=525911ffb58076f44a1788589abac01d&amp;type=3#MCPToolsWindow" />
```

Comment:
I worry about this line, it has hardcoded path the local location, which will be different in production installation.